Mathlib/AlgebraicGeometry/Pullbacks.lean

```diff
@@ -620,4 +620,43 @@ instance Scheme.pullback_map_isOpenImmersion {X Y S X' Y' S' : Scheme}
   rw [pullback_map_eq_pullbackFstFstIso_inv]
   infer_instance
 
+section Spec
+
+variable (R S T : Type u) [CommRing R] [CommRing S] [CommRing T] [Algebra R S] [Algebra R T]
+
+open TensorProduct Algebra.TensorProduct CommRingCat RingHomClass
+
+/-- The isomorphism between the fiber product of two affine schemes `Spec B` and `Spec C`
+over an affine scheme `Spec A` and the `Spec` of the tensor product `B ⊗[A] C`.-/
+noncomputable
+def pullbackSpecIso :
+    pullback (Spec.map (CommRingCat.ofHom (algebraMap R S)))
+      (Spec.map (CommRingCat.ofHom (algebraMap R T))) ≅ Spec (.of <| S ⊗[R] T) :=
+  letI H := IsLimit.equivIsoLimit (PullbackCone.eta _)
+    (PushoutCocone.isColimitEquivIsLimitOp _ (CommRingCat.pushoutCoconeIsColimit R S T))
+  limit.isoLimitCone ⟨_, isLimitPullbackConeMapOfIsLimit Scheme.Spec _ H⟩
+
+@[reassoc (attr := simp)]
+lemma pullbackSpecIso_inv_fst :
+    (pullbackSpecIso R S T).inv ≫ pullback.fst _ _ = Spec.map (ofHom includeLeftRingHom) :=
+  limit.isoLimitCone_inv_π _ _
+
```

Comment:
/--
The composition of the inverse of the isomorphism `pullbackSepcIso R S T` (from the pullback of
`Spec S ⟶ Spec R` and `Spec T ⟶ Spec R` to `Spec (S ⊗[R] T)`) with the second projection is
the morphism `Spec (S ⊗[R] T) ⟶ Spec S` obtained by applying `Spec.map` to the ring morphism
`t ↦ 1 ⊗ₜ[R] t`.